src/app/event/tabs/environment.tpl.html

```diff
@@ -1,59 +1,59 @@
 <h3 class="visible-print">Environment</h3>
 <table class="table table-striped table-bordered table-fixed table-key-value b-t">
   <tr>
-    <th>Occurred On</th>
-    <td>{{::vm.event.date | date: 'medium'}} (<timeago date="vm.event.date"></timeago>)</td>
+    <th>{{'Occurred On' | translate}}</th>
+    <td>{{'DateTime' | translate:vm.event}}(<timeago date="vm.event.date"></timeago>)</td>
```

Comment:
I see in the translation file now that it has this binding. I'd kind of like to revert any date translation changes for now, because it isn't straight forward how the date is being formatted or what property is being formatted/translated. Ideally, long term we would make a user profile setting that allowed you to pick your date format.

Reply:
@Varorbc looking at this again, We'd feel more comfortable if this and a few other changes were reverted (we can talk about them more in a new issue).